app/pages/preview/ui.test.js

```diff
@@ -210,25 +210,21 @@ test('Solution description section and all questions should be rendered', async
   await pageSetup(t, true);
 
   const solutionDescriptionSection = Selector('[data-test-id="view-solution-description"]');
-  const summaryQuestion = solutionDescriptionSection.find('[data-test-id="view-section-question-summary"]');
-  const descriptionQuestion = solutionDescriptionSection.find('[data-test-id="view-section-question-description"]');
-  const linkQuestion = solutionDescriptionSection.find('[data-test-id="view-section-question-link"]');
+  const summaryQuestion = solutionDescriptionSection.find('[data-test-id="view-section-table-row-summary"]');
+  const descriptionQuestion = solutionDescriptionSection.find('[data-test-id="view-section-table-row-description"]');
 
   await t
     .expect(solutionDescriptionSection.exists).ok()
     .expect(await extractInnerText(solutionDescriptionSection.find('h3'))).eql('Description')
 
     .expect(summaryQuestion.exists).ok()
-    .expect(await extractInnerText(summaryQuestion.find('[data-test-id="view-question-title"]'))).eql('Summary')
+    .expect(await extractInnerText(summaryQuestion.find('[data-test-id="view-section-table-row-title"]'))).eql('Summary')
     .expect(await extractInnerText(summaryQuestion.find('[data-test-id="view-question-data-text-summary"]'))).eql('The solution summary')
 
     .expect(descriptionQuestion.exists).ok()
-    .expect(await extractInnerText(descriptionQuestion.find('[data-test-id="view-question-title"]'))).eql('Full description')
+    .expect(await extractInnerText(descriptionQuestion.find('[data-test-id="view-section-table-row-title"]'))).eql('Full description')
     .expect(await extractInnerText(descriptionQuestion.find('[data-test-id="view-question-data-text-description"]'))).eql('The solution description')
-
-    .expect(linkQuestion.exists).ok()
-    .expect(linkQuestion.find('[data-test-id="view-question-title"]').exists).notOk()
-    .expect(await extractInnerText(linkQuestion.find('[data-test-id="view-question-data-link"]'))).eql('The solution link');
+    .expect(await extractInnerText(descriptionQuestion.find('[data-test-id="view-question-data-link"]'))).eql('The solution link');
```

Comment:
can you check the link href?

Reply:
the link is the same as the text, it's probably just a bit confusing because the link in this case would take you to 'The solution link'

Reply:
I see what you mean but can we still check the href?